python/src/pre_recorded/multilingual_librispeech_benchmark.py

```
@@ -0,0 +1,622 @@
+        f"{'Time':>6} | {'Transcription (first 80 chars)'}"
+    )
+    print("\n" + "=" * 160)
+    print(header)
+    print("-" * 160)
+    for r in results:
+        langs_str = ",".join(r.languages) if r.languages else ""
+        code_switching_str = "" if r.code_switching is None else str(r.code_switching)
+        if r.error:
+            print(
+                f"{r.test_case:<40} | {'ERROR':>10} | {langs_str:>14} | "
+                f"{str(r.code_switching or ''):>10} | {'':>12} | "
```

Ruff (0.15.5)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)